cmd/glasskube/cmd/list.go

```diff
@@ -109,6 +146,28 @@ func printPackageTable(packages []*list.PackageWithStatus) {
 	}
 }
 
+func printPackageJSON(packages []*list.PackageWithStatus) {
+	enc := json.NewEncoder(os.Stdout)
+	enc.SetIndent("", "    ")
+	err := enc.Encode(packages)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "error marshaling data to JSON: %v\n", err)
+		cliutils.ExitWithError()
+	}
+}
+
+func printPackageYAML(packages []*list.PackageWithStatus) {
+	for _, pkg := range packages {
+		yamlData, err := yaml.Marshal(pkg)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "error marshaling data to YAML: %v\n", err)
+			cliutils.ExitWithError()
+		}
+		fmt.Println(string(yamlData))
+		fmt.Println("---")
```

Comment:
This produces an empty document at the end. The boundary marker should be printed before the `yamlData`. Additionally, I think it would make sense to only print the boundary marker if there are at least two packages.